.changeset/lemon-nails-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/babel-plugin': minor
+---
+
+Fixes destructuring not happening when arrow functions returns property: value pairs. Added support for nested and alias destructuring.
```

Comment:
What about this?

Handle destructuring `property: value` pairs returned from arrow functions, and add support for nested and alias destructuring